src/AWSCore.jl

```diff
@@ -456,7 +462,11 @@ function do_request(r::AWSRequest)
                                "BadDigest",
                                "RequestTimeoutException",
                                "PriorRequestNotComplete") ||
-                  header(e.cause, "crc32body") == "x-amz-crc32") end
+                  header(e.cause, "crc32body") == "x-amz-crc32")
+            if debug_level > 1
+                println("Caught $e in request $(dump_aws_request(r)), retrying due to throttling...")
```

Comment:
BadDigest and CRC32 are not throttling, PriorRequestNotComplete may not be either but I'm not sure about that one